spec/system/jobseekers_can_view_a_job_application_spec.rb

```diff
@@ -57,7 +57,7 @@
       expect(page).to have_content(I18n.t("jobseekers.job_applications.show.shortlist_alert.title"))
 
       within ".timeline-component" do
-        expect(page).to have_content(I18n.t("jobseekers.job_applications.status_timestamps.unsuccessful"))
+        expect(page).to have_content(I18n.t("jobseekers.job_applications.status_timestamps.shortlisted"))
```

Comment:
I think you need to change more than this. See the context description, and line 57, etc.